### DISCUSSION TITLE
Possibility to schedule a publication for an app

### DISCUSSION THREAD

**Discussion by Steven-SBSI** [microsoft/AL-Go#1556] [Ideas]
Hello,
Is there any possibility to schedule a publication for an app as in Dev Ops ?
Thanks.
Steven

**Comment by freddydk** [comment]
Sorry, didn't see this one.
I don't think there is a way run a workflow once at a later time in GitHub at this moment.

You can setup required reviewers on environments and a wait timer, but that is likely not what you are looking for:
![image](https://user-images.githubusercontent.com/10775043/220252392-2526e908-ad58-4986-ba05-7526f0c38b5a.png)
